docs/pages/tutorial/eas/manage-app-versions.mdx

```diff
@@ -71,6 +73,22 @@ In **eas.json**:
 
 When we create a new production build in the next two chapters, the `versionCode` for Android and `buildNumber` for iOS will increment automatically.
 
+<Collapsible summary="Syncing developer-facing app versions for already published apps to EAS">
+
+If your app is already published in the app stores, the developer-facing app versions are already set. When migrating this app to use EAS Build, follow the steps below to sync those app versions:
+
+- In the terminal window, run the `eas build:version:set` command:
+
+<Terminal cmd={['eas build:version:set']} />
+
+- Select the platform (Android or iOS) when prompted.
+- Select **yes** when asked **Do you want to set app version source to remote now?**. This will set the `cli.appVersionSource` to `remote` in **eas.json**.
```

Comment:
```suggestion
- When asked **Do you want to set app version source to remote now?**, select **yes**. This will set the `cli.appVersionSource` to `remote` in **eas.json**.
```

Wording nit.